opendata.swiss/ui/app/components/dataset/OdsMultiSelect.vue

```diff
@@ -74,25 +64,26 @@
 
 <script setup lang="ts">
 import VSelect from 'vue-select'
-import { ref, computed, onMounted, h, watch } from 'vue'
+import { ref, computed, onMounted, h } from 'vue'
 import { v4 as uuidv4 } from 'uuid'
 
-interface Item {
-  id: string;
-  title: string | undefined;
-  count: number;
-}
-
 const selectId = ref('')
-const currentSelected = ref([] as string[])
+const selected = defineModel({
+  type: Array as () => unknown[],
+  default: () => [],
+})
```

Comment:
tl;dr;, by using `v-model` binding between `multi-select` and `filter-panel` made the code simpler